packages/flash/src/index.ts

```diff
@@ -63,9 +67,9 @@ function makeFlashStorage(req: Request): FlashStorage {
   const session = req.session as any;
 
   return {
-    add(type: FlashMessageType, message: string) {
+    add(type: FlashMessageType, message: string | HtmlSafeString) {
       session.flash ??= [];
-      session.flash.push({ type, message });
+      session.flash.push({ type, message: html`${message}`.toString() });
```

Comment:
Random thought: would it be worth it to update `html` so that one could write `html(message).toString()` instead? 🤷

Reply:
I don't think it's worth it, there shouldn't be many cases where this is necessary.